package.json

```diff
@@ -2,46 +2,77 @@
   "name": "@warp-ds/icons",
   "type": "module",
   "version": "0.0.1",
+  "repository": "git@github.com:warp-ds/icons.git",
   "description": "WARP's icon set",
   "main": "index.js",
   "files": [
-    "dist"
+    "dist",
+    "vue",
+    "react",
+    "elements"
   ],
   "exports": {
-    ".": "./index.js"
+    ".": "./index.js",
+    "./react": "./react/index.js",
+    "./vue": "./vue/index.js",
+    "./elements": "./elements/index.js",
+    "./elements/*": "./elements/*.js",
+    "./package.json": "./package.json"
   },
   "scripts": {
     "import": "node ./scripts/import.js",
     "build": "node ./scripts/build.js",
-    "render": "node preview/render.js",
     "dev": "vite",
     "vite:build": "vite build",
```

Comment:
Looks like `vite build` is only used in `"preview"`, so maybe it's not necessary to keep it as a separate script?

Reply:
[da0b941](https://github.com/warp-ds/icons/pull/4/commits/da0b941617db26a10159664a6b7a588aa76c23bf)